templates/csharp/ai-assistant-bot/appsettings.json

```diff
@@ -7,8 +7,10 @@
     }
   },
   "AllowedHosts": "*",
-  "BOT_ID": "$botId$",
-  "BOT_PASSWORD": "$bot-password$",
+  "BOT_ID": "",
+  "BOT_PASSWORD": "",
+  "BOT_TYPE": "",
+  "BOT_TENANT_ID": "",
```

Comment:
Any change to *appsettings.TestTool.json*?

Reply:
No config is needed for running test tool right? If so I think appsettings.TestTool.json can remain unchanged.